keepercommander/commands/recordv3.py

```diff
@@ -1463,15 +1464,23 @@ def execute(self, params, **kwargs):
 
         recordv3.RecordV3.validate_access(params, record_uid)
         rec = api.get_record(params, record_uid)
-        txt = rec.login if kwargs.get('login') else rec.password
+        if kwargs.get('login'):
+            copy_item = 'Login'
+            txt = rec.login
+        elif kwargs.get('copy_uid'):
+            copy_item = 'Record UID'
+            txt = record_uid
+        else:
+            copy_item = 'Password'
+            txt = rec.password
         if txt:
             if kwargs['output'] == 'clipboard':
                 import pyperclip
                 pyperclip.copy(txt)
-                logging.info('Copied to clipboard')
+                logging.info(f'{copy_item} copied to clipboard')
             else:
-                print(txt)
-            if not kwargs.get('login'):
+                print(f'{copy_item}:\n{txt}')
```

Comment:
I think this change may break customer's script that rely on existing output.
It was a customer's request to dump the password to stdout in order to use it in shell pipeline.

Reply:
I removed the extra text from the copy to stdout.